kroxylicious-api/pom.xml

```diff
@@ -39,11 +39,6 @@
             <groupId>com.github.spotbugs</groupId>
             <artifactId>spotbugs-annotations</artifactId>
         </dependency>
-
-        <dependency>
-            <groupId>io.micrometer</groupId>
-            <artifactId>micrometer-core</artifactId>
```

Comment:
might be worth a note in the changelog that the API had a dependency removed, similar with the kroxylicious-kms deps.